_includes/program-tools.html

```diff
@@ -147,21 +147,21 @@ <h4 class="h4">ClearlyDefined</h4>
     
     <h4 class="h4">CLA Assistant</h4>
```

Comment:
With this latest change, we've actually decided to move away from using the CLA Assistant open source project in favor of this newer solution.

I would update this to cut the part about SAP, point to the updated repo, and instead link to this newer repo as "Microsoft's Contributor License Agreement" or something mor generic like that.

And while I'm proud of our contributions in the past, I would remove the "Microsoft has contributed to the CLA Assistant" since it's no longer the primary focus of this section.

Finally, since this new system is not the CLA Assistant, I would remove that last paragraph.